central_web_server/handle_database.js

```diff
@@ -0,0 +1,19 @@
+const mysql = require('mysql');
+
+const config = require("../database/config.json");
+
+var addSurveyResults = async function(results) {
+	var connection = mysql.createConnection(config.connection);
```

Comment:
you use var here and let for similar things in the same code block - why?

Reply:
Because `let` is scope specific. This means that I couldn't access the for loop's `let item` outside of the loop.

We'll change line 6 to be `let`.

Line 5 can stay as `var` as it is global.

`const` means it is fixed and cannot be changed.